modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ObjectDataContainer.tsx

```diff
@@ -54,7 +60,11 @@ export function ObjectDataContainer({
 			/>
 
 			<InputLocalized
-				disabled={isReadOnly || !hasUpdateObjectDefinitionPermission}
+				disabled={
```

Comment:
@igor-franca Same comment as above.

Reply:
@igor-franca  You can create a const for `!hasUpdateObjectDefinitionPermission || isLinkedNode` to at least extract part of the verification.